src/script/page/migration/AcceptInvitation.tsx

```diff
@@ -54,7 +56,15 @@ export const AcceptInvitation = () => {
   const code = searchParams.get(QUERY_KEY.TEAM_CODE);
   const cachedCode = getTeamInvitationCode();
 
+  const handleEvent = (step: SegmentationValue) => {
+    reportEvent(EventName.USER_MIGRATION_LOGIN, {
+      [SegmentationKey.STEP]: step,
+    });
+  };
```

Comment:
a minor thing, but imo the function name could be more self-descriptive, I would name it something like `trackEvent`

Reply:
i've seen you have this name in other file also